docs/modules/ROOT/pages/explanations/decisions/admin-kubeconfig.adoc

```diff
@@ -0,0 +1,99 @@
+= Admin kubeconfig management
+
+== Problem
+
+We currently store the kubeconfig for the `system:admin` user which is generated by the `openshift-install` program in Passbolt for emergency access to clusters.
+The client certificate generated for that kubeconfig has a lifetime of 10 years.
+Unfortunately, Kubernetes doesn't support revoking client certificates, see https://github.com/kubernetes/kubernetes/issues/18982[this GitHub issue (kubernetes/kubernetes#18982)].
+
+We would like to have another form of emergency access to OpenShift 4 clusters.
+The main reason is that having credentials with a lifetime of 10 years which can't be revoked is less than ideal.
+
+=== Goals
+
+* Define a method to manage emergency access credentials for OpenShift 4 clusters
+* The credentials should be relatively short-lived and it must be possible to rotate them
+
+=== Non-Goals
+
+* Replace regular authentication
+
+== Proposals
+
+=== Credential type
```

Comment:
We have two approaches for "Credentail type" and three approaches for "Credential managment" correct?

Reply:
```suggestion
=== Credential type

For the credential type there are two approaches.
```

Short introduction below the section heading helps the reader to be prepared to  the following paragraphs.

Reply:
I've updated the text to have an overall introduction in section `Proposals`, with short sentences reiterating the intro for each sub-section